integration-manifest.yml

```diff
@@ -0,0 +1,7 @@
+memory: 256M
```

Comment:
This should be covered the CI pipeline.

Reply:
yep, will now remove.. we were testing before concourse was there